staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -644,7 +644,7 @@ type CDISpec struct {
 	// CDIUninstallStrategy defines the state to leave CDI on uninstall
 	UninstallStrategy *CDIUninstallStrategy `json:"uninstallStrategy,omitempty"`
 	// Rules on which nodes CDI infrastructure pods will be scheduled
-	Infra sdkapi.NodePlacement `json:"infra,omitempty"`
+	Infra ComponentConfig `json:"infra,omitempty"`
```

Comment:
This is an api breaking change.  As-is, users with `Infra` set will lose that data on upgrade.  

You could do something like this to be compatible at json serialization level I think.

```golang
type ComponentConfig struct {
  sdkapi.NodePlacement `json:",inline"`
  Replicas *int32 `json:"replicas,omitempty"`
}
```

But I'm not crazy about the name `ComponentConfig`.  Maybe `InfraNodePlacement`?  I'm not sure that simply adding `InfraNodeReplicas *int32` wouldn't be better though

Reply:
Kubevirt also uses a `ComponentConfig` struct for this. Maybe it's a good idea to follow the same naming convention.

Reply:
@alromeros good point I still think it's a bad name but we can use it

Reply:
The name is taken from KubeVirt project, just renamed to `InfraNodePlacement`, thank you for pointing out